analysis/deep2_r23_o32/plotting/te_metal_plots.py

```diff
@@ -29,10 +29,13 @@ def jiang_calibration(metal_det, lo32):
     Parameters
     metal_det -> dictionary of 12+log(O/H)
     lO32 -> log(O32) value
-    """
-    return jiang_coeff[0] + jiang_coeff[1] * metal_det + jiang_coeff[2] * (metal_det * metal_det)\
-           + jiang_coeff[3] * (jiang_coeff[4] + metal_det) * lo32
 
+    Jiang Calibration:
+    jiang_coeff[0] + jiang_coeff[1] * metal_det + jiang_coeff[2] * (metal_det * metal_det) \
+    + jiang_coeff[3] * (jiang_coeff[4] + metal_det) * lo32
+    """
+    return np.poly1d[jiang_coeff[2], jiang_coeff[1], jiang_coeff[0]] \
+            + jiang_coeff[3] * (jiang_coeff[4] + metal_det) * lo32
```

Comment:
I think there's an issue here `np.poly1d`.  See these example:
https://numpy.org/doc/stable/reference/generated/numpy.poly1d.html

Test out and make sure you get the same outputs.

Calling `np.poly1d` creates an object to use.